doc/development_process/release_process.rst

```diff
@@ -151,15 +151,105 @@ The following syntax should be used for releases and tags in Git:
 Long Term Support (LTS)
 =======================
 
-Long-term support releases are designed to be supported for a longer than normal
-period and will be the basis for products and certification for various usages.
-
-An LTS release is made every 2 years and is branched and maintained
-independently from the mainline tree.
-
-An LTS release will be branched and maintained independently of the mainline
-tree.
-
+Long-term support releases are designed to be supported and maintained
+for an extended period and is the recommended release for
+products and the auditable branch used for certification.
+
+An LTS release is defined as:
+
+- **Product focused**
+- **Extended Stabilisation period**: Allow for more testing and bug fixing
+- **Stable APIs**
+- **Quality Driven Process**
+- **Long Term**: Maintained for an extended period of time (at least 2.5 years)
+  overlapping previous LTS release for at least half a year.
+
+
+Product Focused
++++++++++++++++
+
+Zephyr LTS is the recommended release for product makers with an extended
+support and maintenance which includes general stability and bug fixes,
+security fixes.
+
+An LTS includes both mature and new features. API and feature maturity is
+docmented and tracked. The footprint and scope of mature and stable APIs expands
+as we move from one LTS to the next given users access to bleading edge features
+and new hardware while keeping a stable foundation that evolves over time.
+
+Extended Stabilisation Period
++++++++++++++++++++++++++++++
+
+Zephyr LTS development cycle differs from regular releases and has an extended
+stabilization period. Feature freeze of regular releases happens 3-4 weeks
+before the scheduled release date. The stabilisation period for LTS is extended
+by 3 weeks with the feature freeze happening 6-7 weeks before the anticipated
+release date. The time between code freeze and release date is extended in this case.
+
+Stable APIs
+++++++++++++
+
+Zephyr LTS provides a stable and long-lived foundation for developing
+products. To guarantee stability of the APIs and the implemementation of such
+APIs, it is required that any release software that makes the core of the OS
+went through the Zephyr API lifecycle and stabilised over at least 2 releases.
+This guarantees that we release many of the highlighted and core feature with
```

Comment:
features

Also the syntax in the last sentence is not clear to me; what is "implementation supports by stable APIs"?